tests/tools/python/file_checker.py

```diff
@@ -19,7 +19,7 @@
 
 
 def __main__():
-    # ----------------------------------------------------------------arg parsing
+    # ---------------------------------------------------------------arg parsing
```

Comment:
Any reason for these changes?

Reply:
I think with the python code formatting there was a whitespace added at the front and then the comment did not fit the line anymore